packages/ember-routing/lib/services/router.js

```diff
@@ -6,8 +6,33 @@ import { Service, readOnly } from 'ember-runtime';
 import { shallowEqual, resemblesURL, extractRouteArgs } from '../utils';
 
 /**
-   The Router service is the public API that provides component/view layer
-   access to the router.
+   The Router service provides components, and other services, access
```

Comment:
I would remove the `components, and other services` as you can inject any service in any object that has the owner set.